config.py

```diff
@@ -21,7 +21,6 @@ class Config(object):
     SECURITY_USER_PASSWORD = os.getenv("SECURITY_USER_PASSWORD")
     BASIC_AUTH = (SECURITY_USER_NAME, SECURITY_USER_PASSWORD)
     JWT_SECRET = os.getenv("JWT_SECRET")
-    VALIDATE_JWT = os.environ.get("VALIDATE_JWT", True)
```

Comment:
I can't see a valid reason why we ever not want to this this. Firstly VALIDATE_JWT is now a bad name for this as all this is actually doing is checking the expire_in date in the payload. However there is no situation we don't want to do this, in fact it is fairly crucial we do. If we accidentally changed how expire_in change, we want to know about it, in all environments and tests